.github/workflows/codeql-analysis.yml

```diff
@@ -1,6 +1,15 @@
-name: codeql
+name: "CodeQL"
```

Comment:
Same as https://github.com/docker/login-action/pull/622#discussion_r1369364711 if you don't mind :innocent: